fe/fe-core/src/main/java/com/starrocks/lake/StarOSAgent.java

```diff
@@ -248,41 +250,64 @@ public long getWorkerIdByBackendId(long backendId) {
         }
     }
 
-    public void deleteShards(Set<Long> shardIds) throws DdlException {
-        prepare();
-        try {
-            client.deleteShard(serviceId, shardIds);
-        } catch (StarClientException e) {
-            LOG.warn("Failed to delete shards. error: {}", e.getMessage());
-            throw new DdlException("Failed to delete shards. error: " + e.getMessage());
-        }
-    }
-
-    public void createShardGroup(long groupId) throws DdlException {
+    public long createShardGroup(long dbId, long tableId, long partitionId) throws DdlException {
         prepare();
         List<ShardGroupInfo> shardGroupInfos = null;
+        Map<String, String> propertyMap = new HashMap<>();
+        propertyMap.put("dbId", String.valueOf(dbId));
```

Comment:
those properties can be filtered by k, v mapping, label will be more appropriated. it is ok if don't want to do list filtering by these properties.